test-data/unit/check-inference.test

```diff
@@ -1403,7 +1403,7 @@ b[{}] = 1
 
 [case testInferDictInitializedToEmptyAndUpdatedFromMethod]
 map = {}
-def add():
+def add() -> None:
```

Comment:
It's unclear if the annotation was omitted intentionally. I'd suggest creating two variants of the test case.